scripts/_aggregate_cells.py

```diff
@@ -0,0 +1,113 @@
+import os
+import argparse 
+from pathlib import Path
+
+import pandas as pd
+from pycytominer.cyto_utils.cells import SingleCells
+
+
+def aggregate_cells(plate, sql_file, metadata_dir, barcode_platemap, strata, image_cols, compression):
+    """ Aggregates cellular image data. Aggregates are convereted into compressed csv files.
+
+    Parameters
+    ----------
+    plate : str
+       plate identifier 
+    sql_file : str
+        path to file containing cell image data
+    metadata_dir : str
+        path to cell metadata diretory
+    barcode_platemap : str
+        path to file containing barcode metadata
+    strata : list[StringType]
+        columns used for grouping aggregate single cells
+    image_cols : list[StringTypes]
+        selected columns from cell image data 
+    compression : str
+        compression algorithm applied on aggregate data
+        
+    Raises
+    ------
+    FileNotFoundError
+        raised if the single cell data file
+    """
+    print("Processing {}.....".format(plate))
+
+    if not os.path.isfile(sql_file):
+        raise FileNotFoundError("Unable to find SQL file")
+
+    sqlite_file = "sqlite:///{}.sqlite".format(plate)
+
+    # Load specific platemap
+    barcode_platemap_df = pd.read_csv(barcode_platemap)
+    platemap = barcode_platemap_df.query(
+        "Assay_Plate_Barcode == @plate"
+    ).Plate_Map_Name.values[0]
+    platemap_file = os.path.join(metadata_dir, "platemap", "{}.csv".format(platemap))
+    platemap_df = pd.read_csv(platemap_file)
+
+    # Prepare sql file for processing
+    ap = SingleCells(
+        sqlite_file, strata=strata, image_cols=image_cols, fields_of_view_feature=[]
+    )
+
+    # Count cells and output
+    print("Counting cells")
```

Comment:
How does snakemake handle print statements?

Reply:
Removed on commit: 67828fb3af04336c529db4eebfd2c0f8f3958e04


> How does snakemake handle print statements?

The print statement is shown through the terminal but it is kinda lost within the `snakemake` stdouts ... 

However, this was used for debugging purposes. 

Would you like me to implement a more practical print statements when conducting these analysis?